BlockEntity/BEFruitpress.cs

```diff
@@ -519,9 +525,12 @@ private void udpateSqueezeRel(RunningAnimation anim)
             float selfHeight = (float)(juiceableLitresTransfered + juiceableLitresLeft) / 10f;
 
             squeezeRel += Math.Max(0, 0.9f - selfHeight);
+            pressSqueezeRel = GameMath.Clamp(squeezeRel, 0.1f, 1f); //Store the value before performing the Min function so we can tell if the screw is touching the mash yet by comparing the two relative squeezes, in case the player releases and cranks back down.

```

Comment:
Store the value before performing the Min function so we can tell if the screw is touching the mash yet by comparing the two relative squeezes, in case the player releases and cranks back down.